chatter/Chatter.swift

```diff
@@ -18,6 +18,37 @@ It most importantly provides the nextMessage and responseForMessage: methods. Wh
 
 You can also provide an image representation of the character you strive to implement using the designated initializer of this class.
 */
+
+
+
+// Implementation of Topics
+enum Topic : String {
+    case none = "none", Science = "Science", Apples = "Apples", Relativity = "Relativity", Physics = "Physics" // Add Topics here
+    var description: String {
+        switch(self) {
+        case .none:
+            return "none"
+        case .Science:
+            return "Science"
+        case .Apples:
+            return "Apples"
+        case .Relativity:
+            return "Relativity"
+        case .Physics:
+            return "Physics"
+        }
+    }
+}
+// Stores actual topic
+var theCurrentTopic = Topic.none
```

Comment:
Nenn's einfach `topic`. Wie gesagt wäre die Variable in einem `Conversation` Struct zusammen mit den `Chatter`n besser aufgehoben.